primus-core/src/main/java/com/bytedance/primus/executor/ExecutorImpl.java

```diff
@@ -253,6 +302,23 @@ public void transition(ExecutorImpl executor, ExecutorEvent event) {
     }
   }
 
+  static class KillingForciblyTransition implements
```

Comment:
Looks the same as KillingTransition

Reply:
Nice catch! I was thinking having a log showing the transition could be helpful, but not insist on it.